tests/test_server.py

```diff
@@ -27,6 +34,7 @@ def test_cli(args, word):
 def client(tmp_path):
     test_db = tmp_path / 'test.db'
     app = server.create_app(test_db.as_posix())
+    app.config["WTF_CSRF_ENABLED"] = False
```

Comment:
Do note that this currently causes the tests to fail.

Reply:
I think this one is needed for tests and had passed earlier with this config. That said I did put back the same line by mistake in `server.py` as i was also testing routes with curl locally, i think it might be the issue.

Reply:
…Isn't the actual problem that CSRF should be [disabled for API forms (or requests) only](https://wtforms.readthedocs.io/en/3.0.x/meta/#wtforms.meta.DefaultMeta.csrf), rather than globally? The API is hardly usable with CSRF (as the API is meant for external use), but it shouldn't be disabled for non-API requests.